courses/models.py

```diff
@@ -416,6 +430,20 @@ def text_id(self):
         """ Gets the courseware_id"""
         return self.courseware_id
 
+    @property
+    def instructors(self):
+        """List instructors for a course run if they are specified in a related CMS page"""
+        if self.course.page is not None:
+            faculty_page = self.course.page.faculty
+        else:
+            return []
+
+        return (
+            [{"name": member.value["name"]} for member in faculty_page.members]
+            if faculty_page is not None
+            else []
+        )
+
```

Comment:
I'd rather this was moved to `courses.models.Course` and `courses.model.CourseRun.instructors` referred to that.